### ISSUE TITLE
There are some mistakes

### ISSUE THREAD

**Issue by dushyantsapre** [vinyluis/Articles#6]
Hi Viny,

I am trying to under KS better through your code. Really appreciate it but I find it confusing because of following reasons:

1. You have import train_test_split library but you have not split the data.
2. You are directly using training data whereas you should be applying ks_test on test data and predicted values.

Let me know if I am wrong. I would really appreciate if you can show an example on a 2-class classification data set.

Thanks
Dushyant

**Comment by vinyluis** [comment]
Hi Dushyant!

About your points:

Yes, on real applications we would need to split the dataset in train and test, fit the model with the train dataset and evaluate both datasets, comparing their results to check for overfitting. However, the goal of this article is not to be a "complete true-to-reality" example, but rather to show how to use KS as you would use any ohter classification metric, and how KS measures class separabilty.

So in real life, be always sure to split your data in train, validation and test subsets, using them accordingly. If you need further guidance, please check Cassie Kozykrov's amazing "Making Friends With Machine Learning" videos, mainly the ["Importance of data splitting" one](https://www.youtube.com/watch?v=L5rA1MsGXtA&list=PLRKtJ4IpxJpDxl0NTvNYQWKCYzHNuy2xG&index=56) - and all the lectures if you have the time.

The example on the [article](https://towardsdatascience.com/evaluating-classification-models-with-kolmogorov-smirnov-ks-test-e211025f5573) already shows KS on a two-class synthetic problem, but I also [have the code for multiclass classification evaluation](https://github.com/vinyluis/Articles/blob/main/Kolmogorov-Smirnov/Kolmogorov-Smirnov%20-%20Multiclass%20Classification.ipynb).

I hope that I could help with your doubts, and if not, please let me know!
